src/main/java/core/basesyntax/FigureArea.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface FigureArea {
+
+    void area();
```

Comment:
```suggestion
    int getArea();
```
lets create method that return value of area.